checks/src/main/java/tests/ThrowsFinally.java

```diff
@@ -0,0 +1,13 @@
+package main.java.tests;
+
+class ThrowInFinally {
+    void test() {
+        try {
+        } finally {
+            throw new RuntimeException();
```

Comment:
RSPEC-1143: Don't throw or return inside a finally (line 13)
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._